nodejs/week1/avg.js

```diff
@@ -0,0 +1,11 @@
+const args = process.argv.slice(2);
+
+const numbers = args.map(Number).filter(num => !isNaN(num));
```

Comment:
I see you decided to ignore the arguments which are not numbers, instead of failing the code. Well implemented. 👏